### PR TITLE
Refactor and add in WIP table

### PR DESCRIPTION
**Pull request by fourseven**
Here we go, first cut with the WIP table and associated data refactors...

Looks like this:
Screen Shot 2022-04-01 at 2 20 51 PM

### CODE REVIEW

File: _data/helpers.js [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+      classification: 'finished',
+    }
+    // other?
+  ];
+};
+
+module.exports.addProgressStatusToPRs = (prs, person) => {
+  return prs.map(pr => {
+    const updatedPR = {...pr};
+    console.log(ourConstants);
```
Comment (chron) [inline _data/helpers.js:48]: Console.log snuck in 👀

**Review by chron — APPROVED** [review]
Nice!

**Comment by chron** [comment]
Checked out the deploy preview too, I guess this title is no longer accurate:

Screen Shot 2022-04-01 at 2 27 15 PM

And the leaderboard seems entirely missing 👀 

Screen Shot 2022-04-01 at 2 27 22 PM

File: _data/helpers.js (hunk 1 above)
Comment (fourseven) [reply _data/helpers.js:48]: Oh snap

**Pull request merged**